examples/features/interceptor/README.md

```diff
@@ -45,7 +45,7 @@ credential has been configured. If not, configure it to use oauth2 with token
 "some-secret-token" as fallback. In our example, we intentionally omit
 configuring the per RPC credential to resort to fallback.
 
-After pre-processing is done, use can invoke the RPC call by calling the
+After pre-processing is done, user can invoke the RPC call by calling the
```

Comment:
"_the_ user", please

Reply:
Change throughout. I've gone through the entire file to make grammar improvements,PTAL.